packages/jaeger-ui/src/utils/tracking/utils.tsx

```diff
@@ -12,14 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import ReactGA from 'react-ga';
+import { WindowWithGATracking } from './ga';
+
+declare let window: WindowWithGATracking;
 
 // eslint-disable-next-line import/prefer-default-export
 export const logTrackingCalls = () => {
```

Comment:
can we move this function inside ga.tsx (and tests)?  I don't see it being used outside, so odd to expose as public, especially since the dataLayer contact is for GA only.

Reply:
in fact, if feels like it could be combined with the gtag() function instead of always calling it explicitly

Reply:
Aight. The combining idea sounds nice. I'll check it out.

Reply:
I have combined the logging with the `gtag` function itself in the latest implementation. This `WindowWithGATracking` is now only defined and used in `ga.tsx`